### PR TITLE
Update note about logger configuration arguments.

### PR DESCRIPTION
**Pull request by cesarbs**
cc @danroth27 @BrennanConroy

### CODE REVIEW

File: aspnet/fundamentals/logging.rst [hunk 1]
```diff
@@ -144,9 +144,7 @@ A LoggerFactory instance can optionally be configured with custom ``FilterLogger
...
-
-In addition to setting the minimum logging level, you can also specify that whether or not to include scope information in the output by setting ``includeScopes`` to ``true``. You can also specify a function to filter the log levels you wish to include (for example ``l => l >= LogLevel.Warning``) or a function to filter based on both log levels and category strings (for example, ``(category,loglevel) => category.Contains("MyController") && loglevel >= LogLevel.Trace``).
+.. note:: You can specify the minimum logging level each logger provider will use when you configure it. For example, the ``AddConsole`` extension method supports an optional parameter for setting its minimum ``LogLevel``. You can also specify a function to filter the log levels you wish to include (for example ``l => l >= LogLevel.Warning``) or a function to filter based on both log levels and category strings (for example, ``(category, loglevel) => category.Contains("MyController") && loglevel >= LogLevel.Trace``). The ``ConsoleLogger` allows you to specify whether or not to include scope information in the output by setting ``includeScopes`` to ``true``.
```
Comment (danroth27) [inline aspnet/fundamentals/logging.rst]: This feels too big a meaty to be a note. I think we should just remove the note and make this a normal paragraph.

Comment (danroth27) [inline aspnet/fundamentals/logging.rst]: > `l => l >= LogLevel.Warning`

Should this just be `l >= LogLevel.Warning`? Can we give the `l` parameter a name so folks know what it represents?

**Comment by Rick-Anderson** [comment]
Nice improvement.  See #1441

File: aspnet/fundamentals/logging.rst (hunk 1 above)
Comment (danroth27) [inline aspnet/fundamentals/logging.rst]: > `l => l >= LogLevel.Warning`

I think `l` is the logLevel, right? Can we just call the parameter that?

**Comment by cesarbs** [comment]
@Rick-Anderson Thanks for the pointer. I've left some feedback there. Let me know what you think and I can write it up.

**Comment by cesarbs** [comment]
I've changed things quite a bit, let me know what you guys think.

**Comment by cesarbs** [comment]
@kichalla

File: aspnet/fundamentals/logging.rst [hunk 2]
```diff
@@ -136,7 +136,17 @@ Scopes are not required, and should be used sparingly, if at all. They're best u
 Configuring Logging in your Application
 ----------------------------------------
 
-To configure logging in your ASP.NET application, you should resolve ``ILoggerFactory`` in the ``Configure`` method in your ``Startup`` class. ASP.NET will automatically provide an instance of ``ILoggerFactory`` using :doc:`dependency-injection` when you add a parameter of this type to the ``Configure`` method. Once you've added ``ILoggerFactory`` as a parameter, you configure loggers within the ``Configure`` method by calling methods (or extension methods) on the logger factory. We have already seen an example of this configuration at the beginning of this article, when we added console logging by simply calling ``loggerFactory.AddConsole``.
+To configure logging in your ASP.NET application, you should resolve ``ILoggerFactory`` in the ``Configure`` method in your ``Startup`` class. ASP.NET will automatically provide an instance of ``ILoggerFactory`` using :doc:`dependency-injection` when you add a parameter of this type to the ``Configure`` method.
```
Comment (BrennanConroy) [inline aspnet/fundamentals/logging.rst]: ASP.NET Core?

Comment (BrennanConroy) [inline aspnet/fundamentals/logging.rst]: in the `Configure` method **of** your `Startup` class.?

**Comment by cesarbs** [comment]
@BrennanConroy Updated.

File: aspnet/fundamentals/logging.rst [hunk 3]
```diff
@@ -136,7 +136,17 @@ Scopes are not required, and should be used sparingly, if at all. They're best u
 Configuring Logging in your Application
 ----------------------------------------
 
-To configure logging in your ASP.NET application, you should resolve ``ILoggerFactory`` in the ``Configure`` method in your ``Startup`` class. ASP.NET will automatically provide an instance of ``ILoggerFactory`` using :doc:`dependency-injection` when you add a parameter of this type to the ``Configure`` method. Once you've added ``ILoggerFactory`` as a parameter, you configure loggers within the ``Configure`` method by calling methods (or extension methods) on the logger factory. We have already seen an example of this configuration at the beginning of this article, when we added console logging by simply calling ``loggerFactory.AddConsole``.
+To configure logging in your ASP.NET Core application, you should resolve ``ILoggerFactory`` in the ``Configure`` method of your ``Startup`` class. ASP.NET will automatically provide an instance of ``ILoggerFactory`` using :doc:`dependency-injection` when you add a parameter of this type to the ``Configure`` method.
```
Comment (BrennanConroy) [inline aspnet/fundamentals/logging.rst]: Another ASP.NET Core :P

**Comment by cesarbs** [comment]
@BrennanConroy Fixed all instances of "ASP.NET" to "ASP.NET Core" in the document.

**Comment by BrennanConroy** [comment]
:shipit:

**Pull request merged**